.github/workflows/canonical-cla-check.yml

```diff
@@ -0,0 +1,10 @@
+name: Canonical CLA check
```

Comment:
we should add this CLA check before we start encouraging community contributions

Reply:
The CLA check action excludes users with Canonical emails from this check.